docs/docs/typography-js.md

```diff
@@ -64,32 +60,38 @@ const typography = new Typography({
 export default typography
 ```
 
-Font sizes of all elements in Typography.js grow and shrink in relation to the `baseFontSize` defined above. Try playing around with this value and see the visual difference it can make to your website.
+Font sizes of all elements in Typography.js grow and shrink in relation to the `baseFontSize` defined above. Try playing around with this value to see the visual difference it can make to your website.
 
-A full list of options that can be specified when defining a new typography can be found at [Typography.js](https://kyleamathews.github.io/typography.js/).
+To find or create a new typography theme, you can visit [Typography.js](https://kyleamathews.github.io/typography.js/) to see a list of options.
 
 ## Installing Typography themes
 
-Typography.js has some built in themes that can save time when defining your websites font styling. The Funston theme maintained by Typography is one such theme and can be installed from npm running the command: `npm install typography-theme-funston --save`
+Typography.js has built in themes that can save time when defining your website's font styling. The Funston theme, maintained by Typography, is one of the built in themes. To install the Funston theme from npm, run the command: `npm install typography-theme-funston --save`
 
-To use the theme, edit the `typography.js` file that you created before and inform typography of the new configuration:
+To use the theme, edit the `typography.js` file that you created before and inform Typography of the new configuration:
 
 ```diff:title=src/utils/typography.js
 import Typography from "typography";
+// highlight-start
 + import funstonTheme from 'typography-theme-funston'
+// highlight-end
 const typography = new Typography(
+// highlight-start
```

Comment:
It looks like this could go on the line before `funstonTheme` on line 86? Since the other configurations here like baseFontSize were included previously?

Reply:
I wanted to highlight all the code changes, additions and subtractions. Is it more intuitive if we only highlight the additions?

Reply:
Yup, that makes sense! Just made the changes. We could always change it later if it turns out people aren't understanding the highlights for code additions/subtractions.

Let me know if there's anything else of concern!